### PR TITLE
Mw nehemiah

### PR DESCRIPTION
**Pull request by mwerle**
Tweaks to the metadata for the Nehemiah mods; mainly:
* changed author (mwerle -> Micha)
* updated dependencies and recommendations
* added descriptions

### CODE REVIEW

File: NetKAN/KeminiResearchProgram.netkan [hunk 1]
```diff
@@ -5,15 +5,18 @@
...
-    "author": [ "N3h3miah", "mwerle" ],
+    "description": "The Kemini Research Program is a set of small orbital experiments inspired by real experiments run during the Gemini missions. No additional equipment is required - simply install an experiment in the capsule and run it once in orbit."
+    "author": [ "N3h3miah", "Micha" ],
     "license": "GPL-3.0",
     "resources": {
-        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/149298-13-nehemia-engineering-orbital-science-neos-was-oms/"
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?showtopic=149298"
     },
     "depends": [
+        { "name": "ModuleManager" }
```
Comment (Olympic1) [inline NetKAN/KeminiResearchProgram.netkan]: Add a comma

File: NetKAN/KeminiResearchProgram.netkan [hunk 2]
```diff
@@ -5,15 +5,18 @@
...
         { "name": "NehemiahScienceCommon", "min_version": "1:0.7b2" }
     ],
     "recommends": [
+        { "name": "Corvus125mTwoKerbalPod" }
```
Comment (Olympic1) [inline NetKAN/KeminiResearchProgram.netkan]: Add a comma

File: NetKAN/KerbalEnvironmentalEffectsStudy.netkan [hunk 3]
```diff
@@ -5,18 +5,20 @@
...
-        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/149298-13-nehemia-engineering-orbital-science-neos-was-oms/"
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?showtopic=149298"
     },
     "depends": [
-        { "name": "NehemiahScienceCommon", "min_version": "1:0.7b2" },
         { "name": "KIS" }
+        { "name": "NehemiahScienceCommon", "min_version": "1:0.7b2" },
     ],
     "recommends": [
+        { "name": "Corvus125mTwoKerbalPod" }
```
Comment (Olympic1) [inline NetKAN/KerbalEnvironmentalEffectsStudy.netkan]: Add a comma

File: NetKAN/KerbalEnvironmentalEffectsStudy.netkan [hunk 4]
```diff
@@ -5,18 +5,20 @@
...
         { "name": "K2CommandPodCont" },
-        { "name": "ConnectedLivingSpace" }
+        { "name": "UniversalStorage" },
```
Comment (Olympic1) [inline NetKAN/KerbalEnvironmentalEffectsStudy.netkan]: Remove comma

File: NetKAN/NehemiahInc-Complete.netkan [hunk 5]
```diff
@@ -5,19 +5,21 @@
...
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?showtopic=149298"
     },
     "depends": [
         { "name": "ModuleManager" },
         { "name": "KIS" }
     ],
     "recommends": [
+        { "name": "ConnectedLivingSpace" }
+        { "name": "Corvus125mTwoKerbalPod" }
+        { "name": "DockingPortAlignmentIndicator" }
```
Comment (Olympic1) [inline NetKAN/NehemiahInc-Complete.netkan]: 3x comma

File: NetKAN/NehemiahInc-Complete.netkan [hunk 6]
```diff
@@ -5,19 +5,21 @@
...
         { "name": "K2CommandPodCont" },
         { "name": "UniversalStorage" },
```
Comment (Olympic1) [inline NetKAN/NehemiahInc-Complete.netkan]: Remove comma

File: NetKAN/NehemiahMultiPurposeParts.netkan [hunk 7]
```diff
@@ -4,19 +4,16 @@
...
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?showtopic=149298"
     },
-    "depends": [
-        { "name": "ModuleManager" }
-    ],
     "recommends": [
-        { "name": "K2CommandPodCont" },
-        { "name": "UniversalStorage" },
         { "name": "ConnectedLivingSpace" }
+        { "name": "DockingPortAlignmentIndicator" }
```
Comment (Olympic1) [inline NetKAN/NehemiahMultiPurposeParts.netkan]: 2x comma

File: NetKAN/NehemiahMultiPurposeParts.netkan [hunk 8]
```diff
@@ -4,19 +4,16 @@
...
+        { "name": "UniversalStorage" },
```
Comment (Olympic1) [inline NetKAN/NehemiahMultiPurposeParts.netkan]: Remove comma

File: NetKAN/KerbalEnvironmentalEffectsStudy.netkan [hunk 9]
```diff
@@ -5,18 +5,20 @@
...
+    "description": "Kerbal Environmental Effects Study is a mid-game set of material exposure experiments aimed at small orbital stations. To run an experiment, you have to attach an Experiment Carrier to the outside of your station (using KIS), then add an experiment to it. The idea is to run experiments in the background while doing other missions."
+    "author": [ "N3h3miah", "Micha" ],
     "license": "GPL-3.0",
     "resources": {
-        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/149298-13-nehemia-engineering-orbital-science-neos-was-oms/"
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?showtopic=149298"
     },
     "depends": [
-        { "name": "NehemiahScienceCommon", "min_version": "1:0.7b2" },
         { "name": "KIS" }
```
Comment (Olympic1) [inline NetKAN/KerbalEnvironmentalEffectsStudy.netkan]: Add a comma

File: NetKAN/KerbalEnvironmentalEffectsStudy.netkan [hunk 10]
```diff
@@ -5,18 +5,20 @@
...
+        { "name": "NehemiahScienceCommon", "min_version": "1:0.7b2" },
```
Comment (Olympic1) [inline NetKAN/KerbalEnvironmentalEffectsStudy.netkan]: Remove comma

**Comment by mwerle** [comment]
Updated commas (including some you missed :) ) - thanks for double-checking.

I also added a script to check whether files are valid JSON or not. Could be useful but feel free to ignore (requires bash and python).

**Comment by mwerle** [comment]
Actually before you merge this set of changes, I'm thinking of removing the individual package releases of this mod.  Seeing all the mods in the CKAN mod list it seems rather spammy, especially as some of the sub-packages are just internal.  Ditto in AVC.

This would also clarify the name of "Nehemiah Inc - Complete Pack" to "Nehemiah Engineering Orbital Science".

Comments/suggestions around this?

**Comment by politas** [comment]
If NehemiahInc-Complete is simply all of the otherwise-available NehemiahInc Mods, it would be better to make it a metapackage (Just a set of depends with no installation. See [RealismOverhaul](https://raw.githubusercontent.com/KSP-RO/RealismOverhaul/master/RealismOverhaul.netkan) for an over-the-top example of what can be done). Also, please remove the JSON checking script. This PR will never pass Jenkins with it included.

**Pull request merged**